tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml

```diff
@@ -0,0 +1,13 @@
+---
+__FORKDELTA_CUSTOM_SYMBOL: AVOID CONFLICT WITH 0xd9a0658b7cc9ec0c57e8b20c0920d08f17e747be
+addr: '0xc56b13ebbcffa67cfb7979b900b736b3fb480d78'
+decimals: 8
+description: >-
+  Decentralized social network
+  You take back power with every social connection
```

Comment:
Can we find a better description here?

Reply:
I tried to find some, but as they've launched it seems they let the platform talk for itself.
My wish as well